test/runtests.jl

```diff
@@ -19,30 +19,59 @@ img_eps = isinteractive() ? 1e-2 : 10e-2
 #     image_comparison_facts(:gadfly, skip=[4,6,23,24,27], eps=img_eps)
 # end
 
+facts("GR") do
+    @fact gr() --> Plots.GRBackend()
+    @fact backend() --> Plots.GRBackend()
+
+    image_comparison_facts(:gr, eps=img_eps)
+end
+
 facts("PyPlot") do
     @fact pyplot() --> Plots.PyPlotBackend()
     @fact backend() --> Plots.PyPlotBackend()
 
-    image_comparison_facts(:pyplot, skip=[6,25,30,31], eps=img_eps)
+    image_comparison_facts(:pyplot, skip=[2,31], eps=img_eps)
```

Comment:
I think 25 and 30, at least, depend on StatPlots (25 uses a DataFrame and 30 a violin plot), so they should be skipped, right? 

Also, It'd be nice to keep a record of which are skipped because they take time, and which aren't compatible with a given backend (so we can include those when processing examples for the documentation site).